examples/mod-data-import-converter-storage/jobProfileCollection.sample

```diff
@@ -210,9 +194,7 @@
       "id": "b32e79bc-01d9-4d31-bc08-a3621fcfc1aa",
       "name": "Load EDI Invoice",
       "description": "",
```

Comment:
Why empty ?

Reply:
It's OK, description is an optional field, it can be empty